tests/pytest/pytest_sample/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
```

Comment:
move this under tests/testsuite/pytest

Reply:
Can't push to youhua's repo any more, I update this PR to #33227